recipes/trailrunner/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "trailrunner" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trailrunner-{{ version }}.tar.gz
+  sha256: 157de2b2d82139d9583ec1dbc1981c33b39e24c420bb5294f86083f1f4b44311
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - flit >=3,<4
+    - pathspec >=0.8.1
+  run:
+    - python
+    - pathspec >=0.8.1
```

Comment:
This package has a dependency on `futures3` for py<37. You can either add that, or maybe try to make it `noarch` with a skip for `py<37`

Reply:
I will go for the noarch + skip option. Thanks for the suggestion.

Reply:
Fixed